offline_signing/transfer.sh

```diff
@@ -180,23 +200,23 @@ esac
 # Walk argument pairs.
 while [[ $# > 0 ]]; do
     c="$1"
-    v="${2?Must provide a version (e.g. 1234.0.0)}"
+    v="${2?Must provide an argument for each channel (see usage)}"
     shift 2
 
     case "${c}" in
-    -a)
-        $CMD "alpha" "${v}" "amd64-usr"
-        $CMD "alpha" "${v}" "arm64-usr"
-        ;;
-    -b)
-        $CMD "beta" "${v}" "amd64-usr"
-        $CMD "beta" "${v}" "arm64-usr"
-        ;;
-    -s)
-        $CMD "stable" "${v}" "amd64-usr"
-        ;;
-    *)
-        usage
-        ;;
+        -a)
```

Comment:
Please don't mix in random indentation changes.  The indentation that was there is traditional for bash `case` statements.

Reply:
sorry, that was unintended, my editor just did that for me and the rebasing made it slip through the cracks.